docs/FAQ.rst

```diff
@@ -356,6 +443,20 @@ If you are behind a corporate proxy, make sure you have ``proxy type`` and
 ``aesm proxy`` lines set in ``/etc/aesmd.conf`` .
 This file may be overwritten if you reinstall Intel SGX SDK.
 
+How do I know I am in Intel SGX Hardware Mode?
+----------------------------------------------
+If you set ``SGX_MODE=HW`` in your environment and setup Intel SGX correctly,
+Avalon will startup in Intel SGX Hardware Mode.
+You know you are in Intel SGX Hardware Mode (SGX_MODE=HW) when you see messages
```

Comment:
Also, in case of HW mode we see lengthy worker related details on the console. This is because, in HW mode proof data is obtained from the enclave and its empty in case of SIM mode.